pkg/cli/fixtures_test.go

```diff
@@ -162,23 +162,25 @@ func fxRegistry() *structs.Registry {
 
 func fxRelease() *structs.Release {
 	return &structs.Release{
-		Id:       "release1",
-		App:      "app1",
-		Build:    "build1",
-		Env:      "FOO=bar\nBAZ=quux",
-		Manifest: "services:\n  web:\n    build: .",
-		Created:  time.Now().UTC().Add(-49 * time.Hour),
+		Id:          "release1",
+		App:         "app1",
+		Build:       "build1",
+		Env:         "FOO=bar\nBAZ=quux",
+		Manifest:    "services:\n  web:\n    build: .",
+		Created:     time.Now().UTC().Add(-49 * time.Hour),
+		Description: "description1",
 	}
 }
 
 func fxRelease2() *structs.Release {
 	return &structs.Release{
-		Id:       "release2",
-		App:      "app1",
-		Build:    "build1",
-		Env:      "FOO=bar\nBAZ=quux",
-		Manifest: "manifest",
-		Created:  time.Now().UTC().Add(-49 * time.Hour),
+		Id:          "release2",
+		App:         "app1",
+		Build:       "build1",
+		Env:         "FOO=bar\nBAZ=quux",
+		Manifest:    "manifest",
+		Created:     time.Now().UTC().Add(-49 * time.Hour),
+		Description: "description2",
```

Comment:
Sorry I should have been more clear. Can you please set this one back to blank so we can test both cases (blank description vs one with data)